src/alt/index.js

```diff
@@ -185,20 +185,48 @@ class Alt {
   // Instance type methods for injecting alt into your application as context
 
   addActions(name, ActionsClass, ...args) {
+    if (typeof name !== 'string') utils.warn(`actions must have name`)
+
+    if (Object.is(ActionsClass, undefined) ||
```

Comment:
can we configure this so that you can use envify to remove this from production builds?